tutorials/xarray_tutorial.py

```diff
@@ -6,295 +6,344 @@
 ==========================
 
 `xarray <http://xarray.pydata.org/>`_ is a powerful Python package that provides N-dimensional
-labeled arrays and datasets following the Common Data Model. While the process of integrating
-xarray features into MetPy is ongoing, this tutorial demonstrates how xarray can be used
-within the current version of MetPy. MetPy's integration primarily works through accessors
-which allow simplified projection handling and coordinate identification. Unit and calculation
-support is currently available in a limited fashion, but should be improved in future
-versions.
+labeled arrays and datasets following the Common Data Model. MetPy's suite of meteorological
+calculations are designed to integrate with xarray DataArrays as one of its two primary data
+models (the other being Pint Quantities). MetPy also provides DataArray and Dataset
+*accessors* (collections of methods and properties attached to the ``.metpy`` property) for
+coordinate/CRS and unit operations.
+
+Full information on MetPy's accessors is available in the `reference guide
```

Comment:
Build doesn't like that "reference guide" is used as a target a second time here.